supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,8 +10,52 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use instead: _Month day, year_ (for example, October 3, 2019).
 
 
-[[user-replaced-variables]]
-== User-replaced variables
+[[user-replaced-values]]
+== User-replaced values
+
+A _user-replaced value_, or _replaceable value_, is a value that the user must replace with a value that is relevant for their situation. User-replaced values are often found in places such as code blocks, file paths, and commands.
```

Comment:
@bergerhoffer 
1. Generally only use one term, unless there's a good reason to use more than one. Since you're defining the terms here, is there any reason to include both terms _user-replaced value_ and _replaceable value_?
2. Please add a short example. Suggestion:
```suggestion
A _user-replaced value_  is a value that the user must replace. For example, in the following command `_filename_` and `_destination_` are user-replaced values.

`cp _filename_ _destination_`

User-replaced values are often found in places such as code blocks, file paths, and commands.
```

Reply:
@stoobie I updated to remove the second term mention of "replaceable value". 

The examples are further down after we explain the guidelines. Are those not sufficient?